spec/index.bs

```diff
@@ -3910,7 +3910,8 @@ enum GPUTextureDimension {
     : <dfn>"3d"</dfn>
     ::
         Specifies a texture that has a width, height, and depth. {{GPUTextureDimension/"3d"}}
-        textures cannot be multisampled or use depth/stencil formats.
+        textures cannot be multisampled or use depth/stencil formats,
+        or use compressed formats without [[#packed-formats|sliced 3D support]].
```

Comment:
Slightly easier to read by reducing the amount of negations. This is more documentation than spec, so I like to spell it out in a developer-friendly way. Maybe this would be good - no one probably specifically cares that depth/stencil textures can't be 3d.

```suggestion
        textures cannot be multisampled, and their format must support 3d textures
        (all [[#plain-color-formats]] and some [[#packed-formats|packed/compressed formats]]).
```